cmd/xlsx.go

```diff
@@ -7,7 +7,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/tealeg/xlsx"
+	"github.com/tealeg/xlsx/v2"
```

Comment:
For this to work I believe there ought to be an associated change in `gocsv/go.sum` and `gocsv/cmd/go.sum`.

Currently those entries are:
```
github.com/tealeg/xlsx v0.0.0-20161026161224-a8490cf686de h1:PC5Hwqy6Muk6JOroE699iNIxzZLlnls3iEDXbClQsKc=
```
This means that it is using commit `a8490cf686de` of that repo: tealeg/xlsx@a8490cf686de

That is a pretty old commit (over 3 years old!) so it probably does make sense to upgrade that dependency, but I don't think this quite does it.

Reply:
Hi! Thanks for reviewing!

This PR is definitely broken, as xlsx has all changed since the reorganization for Version 3.x.x.

I'll see if I can push a new change that actually works again.